WooCommerce/Classes/Extensions/SitePlugin+Woo.swift

```diff
@@ -6,6 +6,6 @@ extension SitePlugin {
     enum SupportedPlugin {
         public static let WCShip = "WooCommerce Shipping &amp; Tax"
         public static let WCTracking = "WooCommerce Shipment Tracking"
-        public static let WCSubscriptions = "WooCommerce Subscriptions"
+        public static let WCSubscriptions = ["WooCommerce Subscriptions", "Woo Subscriptions"]
```

Comment:
Beyond the hot fix: I'm surprised that there's not a more robust way of identifying a plugin, like some slug or identifier that is unique, internal (not user-facing and subject to design/product) changes, and permanent?